GiphySearch/GifCell.swift

```diff
@@ -6,34 +6,44 @@
 //
 
 import UIKit
+
 class GifCell: UICollectionViewCell {
+    
+    @IBOutlet private var loader: UIActivityIndicatorView!
+    
     var gif: Gif? {
         didSet {
             fetchGif()
         }
     }
     
-    @IBOutlet weak var loader: UIActivityIndicatorView!
-    var gifView: UIImageView = {
-        let v = UIImageView()
-        v.contentMode = .scaleAspectFit
-        return v
+   private var gifView: UIImageView = {
+        let imageView = UIImageView()
```

Comment:
Use descriptive names, you have auto complete to help you. Unless you're writing on mobile device you'll not get away with using `v` instead of `imageView`